presto-jdbc/src/main/java/com/facebook/presto/jdbc/AbstractConnectionProperty.java

```diff
@@ -192,6 +196,29 @@ private StringMapConverter() {}
         }
     }
 
+    protected static final class CustomHeadersConverter
+            implements Converter<Map<String, String>>
+    {
+        private static final CharMatcher PRINTABLE_ASCII = CharMatcher.inRange((char) 0x21, (char) 0x7E);
```

Comment:
this var doesn't seem to be used anywhere.

Reply:
Removed it.